socialhome/streams/app/components/ReactionsBar.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="grid-item-bar d-flex justify-content-start">
+        <slot />
+        <div class="ml-auto grid-item-reactions mt-1">
+            <b-button
```

Comment:
Does the design match the old one? This used to be a `div`.

![selection_415](https://user-images.githubusercontent.com/1174866/29817069-3846d2ec-8cbf-11e7-973c-a71067d2f089.png)

Reply:
Yes, it does. Styling is attached to `.item-reaction` class.